website/docs/r/cis_page_rule.html.markdown

```diff
@@ -0,0 +1,138 @@
+---
+layout: "ibm"
+page_title: "IBM: ibm_cis_page_rule"
+sidebar_current: "docs-ibm-resource-cis-page-rule"
+description: |-
+  Provides a IBM CIS Page Rule resource.
+---
+
+# ibm_cis_page_rule
+
+Provides a IBM CIS Page Rule resource. This resource is associated with an IBM Cloud Internet Services instance and a CIS Domain resource. It allows to create, update, delete page rules of a domain of a CIS instance
+
+## Example Usage
+
+```hcl
```

Comment:
can the example for forwarding url and minify be added as they are different when compared to others

Reply:
minify is not supported yet.

Reply:
but i see minify in code

Reply:
is the resource not supporting or api?

Reply:
I removed the `minify` from validator.

Reply:
If the api supports please add minify feature as well in the resource..

Reply:
API does not support as of now.